contract-tests/tests/test/amazon/base/contract_test_base.py

```diff
@@ -90,6 +90,7 @@ def setUp(self) -> None:
             .with_env("OTEL_METRIC_EXPORT_INTERVAL", "50")
             .with_env("OTEL_AWS_APPLICATION_SIGNALS_ENABLED", "true")
             .with_env("OTEL_METRICS_EXPORTER", "none")
+            .with_env("OTEL_EXPORTER_OTLP_PROTOCOL", "grpc")
```

Comment:
Can we provide an explanation why we need to specify `grpc` or contract tests and not leave it to default?

Reply:
Contract test framework is entirely built around gRPC export (mock collector is a gRPC server), which was the default. Now default is HTTP, so if we leave it as default, tests will all fail.

Reply:
I see. Maybe we should put a comment in the code explaining this.

Reply:
There's lots of configuration here without comment. I don't think this needs to block this PR.